manifests/charts/istio-operator/values.yaml

```diff
@@ -1,3 +1,10 @@
+
+global:
+  # ImagePullSecrets for operator ServiceAccount, list of secrets in the same namespace
+  # to use for pulling any images in pods that reference this ServiceAccount.
+  # Must be set for any cluster configured with private docker registry.
+  imagePullSecrets: []
```

Comment:
Why do you add it to `global` section? I think `imagePullSecrets` should be in top level.

Reply:
it is already so in all the other istio charts. i'm trying to keep everything aligned.